packages/editor-ui/src/components/ParameterInputFull.vue

```diff
@@ -8,7 +8,7 @@
 			</n8n-tooltip>
 		</el-col>
 		<el-col :span="isMultiLineParameter ? 24 : 14" class="parameter-value">
-			<parameter-input :parameter="parameter" :value="value" :displayOptions="displayOptions" :path="path" :isReadOnly="isReadOnly" @valueChanged="valueChanged" inputSize="small" />
+			<parameter-input :parameter="parameter" :value="hasEmptyLoadOptions ? '' : value" :displayOptions="displayOptions" :path="path" :isReadOnly="isReadOnly" @valueChanged="valueChanged" inputSize="small" />
```

Comment:
this only fixes the issue for a narrow case. we should fix it at the level of ParameterInput component itself.. Because we use this component in different places and issue will arise there as well.

it looks to me like when `remoteParameterOptionsLoadingIssues` in `ParameterInput` is not null, it switches to text input.. 
https://github.com/n8n-io/n8n/blob/443c2a4d51882ce975b3f8ba4350dd5d912fe1bb/packages/editor-ui/src/components/ParameterInput.vue#L15
I think that's to show `Loading Options` like in the Typeform Trigger..
<img width="456" alt="Screen Shot 2021-09-22 at 9 55 06 AM" src="https://user-images.githubusercontent.com/4711238/134304713-53a9c97d-1ab4-4a3e-b62b-1f74c11f83f6.png">
So you have a couple of options:
1. Fix it so that it shows loading options in text input
2. Show an empty select and we can change the placeholder values.. Use loading and loadingText properties of selects https://element.eleme.io/#/en-US/component/select#select-attributes
I prefer option 2